src/discord/cogs/system.js

```diff
@@ -4,7 +4,8 @@ const { exec } = require('child_process');
 const r = require('./../../db');
 const i18n = require('i18n');
 const os = require('os');
-const { inspect } = require('util')
+const { inspect } = require('util');
+const snekfetch = require('snekfetch');
```

Comment:
Not in `package.json`, have you tried `request`, which has already been installed?

Reply:
Why would you use request? Snekfetch is a lot faster, and very lightweight.